eZ/Publish/Core/Persistence/Cache/Tags/TagGeneratorInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Cache\Tags;
```

Comment:
Move to the new structure:
```suggestion
namespace Ibexa\Core\Persistence\Cache\Tag;
```